File: src/backend/SchoolBudget/Dto/FundraisingEditDto.cs
namespace SchoolBudget.Dto;

public class FundraisingEditDto
{
    public DateOnly Date { get; set; }
    public string Name { get; set; } = null!;
    public string? Description { get; set; }
    public decimal RecommendedAmount { get; set; }
}

File: src/frontend/src/models/fundraising-edit-dto.ts
import { FundraisingType } from "./fundraising";

export interface FundraisingEditDto {
  id?: number;
  date: Date;
  name: string;
  description: string;
  recommendedAmount: number;
  fundId?: number;
  type?: FundraisingType;
  isClosed?: boolean;
  closingDate?: Date;
}
